packages/desktop/tsconfig.json

```diff
@@ -1,9 +1,14 @@
 {
     "extends": "@tsconfig/svelte/tsconfig.json",
     "compilerOptions": {
-        "isolatedModules": false,
         "allowJs": true,
-        "lib": ["DOM", "ESNext"]
+        "baseUrl": ".",
+        "isolatedModules": false,
+        "lib": ["DOM", "ESNext"],
+        "paths": {
+            "@common": ["../shared/lib/common/*"],
+            "@core": ["../shared/lib/core/*"]
+        }
```

Comment:
Ideally yes, but since that directories houses JS files it wouldn't. I know we've got a migration to TS planned (see [here](https://github.com/iotaledger/firefly/issues/2081)), in which case we could and should add an alias afterwards :+1:.